src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -31,6 +31,11 @@ internal static class UnmanagedMethods
 
         public delegate IntPtr WndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam);
 
+        public static readonly IntPtr DPI_AWARENESS_CONTEXT_UNAWARE = new IntPtr(-1);
```

Comment:
just curious why these are not integers?

Reply:
Yeah it's strange but this just seems to be how they're defined:

https://github.com/mintty/mintty/blob/master/src/winmain.c#L229
https://stackoverflow.com/questions/41708679/how-to-convert-declare-handle-and-subsequent-consts-from-windef-h-to-delphi
https://github.com/AArnott/pinvoke/blob/300c4bf6a9fdc0f82f3b82402313a5253b9c0d78/src/User32/User32.cs#L91